resources/dashboards/Traffic-Endpoints-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "hits": 0, 
+   "timeRestore": false, 
+   "description": "", 
+   "title": "Traffic Endpoints Dashboard", 
+   "panelsJSON": "[{\"col\":10,\"id\":\"Top-Applications-By-Sessions-(pie)\",\"row\":3,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":10,\"id\":\"Top-Applications-By-Metadata-Impact-(pie)\",\"row\":1,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Top-Applications-By-Bandwidth-(pie)\",\"row\":1,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Top-Applications-By-Packet-Count\",\"row\":3,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Top-Applications-By-Sessions-(histogram)\",\"row\":1,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Sum-of-Repeated-Field-Count-by-Application\",\"row\":8,\"size_x\":6,\"size_y\":3,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Average-Repeated-Elements-in-a-Message-(All-Reports)\",\"row\":5,\"size_x\":6,\"size_y\":3,\"type\":\"visualization\"},{\"id\":\"Average-Message-Size-(All-Reports)\",\"type\":\"visualization\",\"size_x\":6,\"size_y\":3,\"col\":7,\"row\":5}]", 
```

Comment:
Some of these visualizations have `(pie)` or `(histogram)` in the title and some do not. Let's make these consistent. It looks like other dashboards have them in the titles, so for those that do not lets add it.